lib/next_rails/bundle_report/ruby_version_compatibility.rb

```diff
@@ -0,0 +1,43 @@
+require "colorize"
+
+class NextRails::BundleReport::RubyVersionCompatibility
+  attr_reader :gems, :options
+
+  def initialize(gems: NextRails::GemInfo.all, options: {})
+    @gems = gems
+    @options = options
+  end
+
+  def generate
+    return invalid_message unless valid?
+
+    message
+  end
+
+  private
+
+  def message
+    output = "=> Incompatible gems with Ruby #{ruby_version}:".white.bold
+    incompatible.each do |gem|
+      output += "\n#{gem.name} - required Ruby version: #{gem.gem_specification.required_ruby_version}".magenta
+    end
+    output += "\n\n#{incompatible.length} incompatible #{incompatible.length == 1 ? 'gem' : 'gems' } with Ruby #{ruby_version}".red
```

Comment:
would it be compatible with oldest ruby versions?

```suggestion
    output += "\n\n#{incompatible.length} incompatible #{incompatible.one? ? 'gem' : 'gems' } with Ruby #{ruby_version}".red
```

Reply:
Do you mean `#one?`? Apparently, [has been available](https://ruby-doc.org/core-1.9.1/Enumerable.html#method-i-one-3F) since `1.9`, but it was on Enumerable (now is on [Array](https://ruby-doc.org/3.2.2/Array.html#method-i-one-3F))